l10n_it_edi_extension/models/account_move.py

```diff
@@ -0,0 +1,874 @@
+            ):
+                self.l10n_it_edi_tax_representative_id = tax_representative.id
+            else:
+                if tax_representative := self._l10n_it_edi_create_partner(
+                    body_tree, "//RappresentanteFiscale", vat, codice_fiscale
+                ):
+                    self.l10n_it_edi_tax_representative_id = tax_representative.id
+
+    def _l10n_it_edi_get_intermediary(self, body_tree):
+        if body_tree.xpath("//TerzoIntermediarioOSoggettoEmittente"):
+            vat = get_text(
+                body_tree, "//TerzoIntermediarioOSoggettoEmittente//IdCodice"
+            )
+            codice_fiscale = get_text(
+                body_tree, "//TerzoIntermediarioOSoggettoEmittente//CodiceFiscale"
+            )
+            self = self.with_context(skip_create_partner=True)
+            if intermediary := self._l10n_it_edi_search_partner(
+                self.company_id, vat, codice_fiscale, ""
+            ):
+                self.l10n_it_edi_intermediary_id = intermediary.id
+            else:
+                if intermediary := self._l10n_it_edi_create_partner(
+                    body_tree,
+                    "//TerzoIntermediarioOSoggettoEmittente",
+                    vat,
+                    codice_fiscale,
+                ):
+                    self.l10n_it_edi_intermediary_id = intermediary.id
+
+        if sender := get_text(body_tree, "//SoggettoEmittente"):
+            self.l10n_it_edi_sender = sender
+
+    def _l10n_it_edi_get_extra_info(
+        self, company, document_type, body_tree, incoming=True
+    ):
+        extra_info, message_to_log = super()._l10n_it_edi_get_extra_info(
+            company, document_type, body_tree, incoming=incoming
+        )
+
+        self._l10n_it_edi_get_tax_representative(body_tree)
+        self._l10n_it_edi_get_intermediary(body_tree)
+
+        if rounding := get_float(body_tree, ".//DatiGeneraliDocumento/Arrotondamento"):
+            self.l10n_it_edi_rounding = rounding
+
+        if get_text(body_tree, "//DatiGeneraliDocumento/Art73"):
+            self.l10n_edi_it_art73 = True
+
+        if elements_sal := body_tree.xpath(".//DatiGenerali/DatiSAL"):
+            for element_sal in elements_sal:
+                self.env["l10n_it_edi.activity_progress"].create(
+                    {
+                        "activity_progress": get_text(
+                            element_sal, ".//RiferimentoFase"
+                        ),
+                        "invoice_id": self.id,
+                    }
+                )
+
+        for xpath, label in [
+            (".//DatiGenerali/DatiTrasporto", "Transport informations from XML file:"),
+            (".//DatiVeicoli", "Vehicle informations from XML file:"),
+        ]:
+            if body_tree.xpath(xpath):
+                message = Markup("<br/>").join(
+                    (self.env._(label), self._compose_info_message(body_tree, xpath))
```

Comment:
```suggestion
            (".//DatiGenerali/DatiTrasporto", self.env._("Transport informations from XML file:")),
            (".//DatiVeicoli", self.env._("Vehicle informations from XML file:")),
        ]:
            if body_tree.xpath(xpath):
                message = Markup("<br/>").join(
                    label, self._compose_info_message(body_tree, xpath))
```
Altrimenti queste stringhe non vengono esportate come da tradurre perché `_` non è abbastanza furbo per capire cosa c'è dentro `label`

Reply:
:+1: